docs/sriov.md

```diff
@@ -123,29 +196,55 @@ Once the cluster is deployed, we can move to SR-IOV specific components.
 
 # Deploy SR-IOV services
 
-First, deploy latest Multus with default Flannel backend. We will need to use
-the latest code from their tree, hence using `snapshot` image tag instead of
-`latest`. The `snapshot` image adds support for reading IDs of devices
-allocated by device plugins from "checkpoint" files, which is needed to make
-the whole setup work.
+First, deploy latest Multus with default Flannel backend.
 
 ```
 $ go get -u -d github.com/intel/multus-cni
 $ cd $GOPATH/src/github.com/intel/multus-cni/
-$ vi images/multus-daemonset.yml # change to refer to nfvpe/multus:snapshot
 $ mkdir -p /etc/cni/net.d
 $ cp images/70-multus.conf /etc/cni/net.d/
 $ ./cluster/kubectl.sh create -f $GOPATH/src/github.com/intel/multus-cni/images/multus-daemonset.yml
 $ ./cluster/kubectl.sh create -f $GOPATH/src/github.com/intel/multus-cni/images/flannel-daemonset.yml
 ```
 
-The best way to configure your SR-IOV devices is by using the [OpenShift SR-IOV operator](https://github.com/openshift/sriov-network-operator).
+Now, deploy SR-IOV device plugin. Adjust config.json file for your particular
```

Comment:
Why you reverted to the "manual" installation process? In any case, I would at least mention the operator as an alternative way to do all the stuff described here

Reply:
I put a similar recommendation to use the operator at the very top of the document, leaving the rest of the document for manual (discouraged) deployment steps. With that, I don't think we should have this note, again, here down below. What do you think?

Reply:
Not sure if it's confusing for a non aware reader. I'd leave a little note under _ Deploy SR-IOV services_ mentioning that is the manual path in case you don't want to use the operator (or for educational purpouses)

Reply:
I'm adding a tiny note.